src/browser-polyfill.js

```diff
@@ -132,7 +132,48 @@ if (typeof browser === "undefined") {
         }
 
         return new Promise((resolve, reject) => {
-          target[name](...args, makeCallback({resolve, reject}, metadata));
+          function callAPIWithNoCallback() {
+            try {
+              target[name](...args);
+              if (chrome.runtime.lastError) {
+                // NOTE: Ideally this should rejects if there are runtime issues with
+                // the API call on Chrome, unfortunately currently when pageAction.show/hide
```

Comment:
This whole comment block will go probably go away because of my above comment. But if you decide to re-instate the comments somewhere, generalize the affected APIs, e.g. "APIs such as pageAction.show" because pageAction.show/hide are not the only affected APIs.